### ISSUE TITLE
G104: _, _ = Write(...)

### ISSUE THREAD

**Issue by fredbi** [golangci/golangci-lint#234]
Local golangci-lint and online disagree on how best to write this:
```golang
rw.Write(errorAsJSON(New(http.StatusInternalServerError, "Unknown error")))
```
With local golangci-lint, this shows up as an error (unchecked error).
```
api.go:143:12: Error return value of `rw.Write` is not checked (errcheck)
```

Ok; so here is the corrected line of code:
```golang
_, _ = rw.Write(errorAsJSON(New(http.StatusInternalServerError, "Unknown error")))
```

> from: https://github.com/go-openapi/errors

I updated golangci-lint to latest master and installed it locally.
This works fine as expected, and no error is reported (no .golangci.yml).

When I am pushing this to the repo running online golangci then I get:
```
 golangcibot 37 minutes ago

G104: Errors unhandled.
```

1. Version of golangci-lint: `golangci-lint --version` (or git commit if you don't use binary distribution)
local run:
git commit: 0c82b641236764940235a15c3d51a4623b8045a1

2. Go environment: `go version && go env`
go1.10.3 linux/amd64

3. Verbose output of running: `golangci-lint run -v`
```
level=info msg="[config_reader] Config search paths: [./ /home/go/src/github.com/go-openapi/errors /home/go/src/github.com/go-openapi /home/go/src/github.com /home/go/src /home/go /home /]"
level=info msg="[config_reader] Used config file .golangci.yml"
level=info msg="[lintersdb] Active 22 linters: [deadcode depguard dupl errcheck goconst gocyclo gofmt goimports golint gosec govet ineffassign interfacer megacheck misspell nakedret prealloc structcheck typecheck unconvert unparam varcheck]"
level=info msg="[load/path_resolver] Skipped dirs: [.git .github]"
level=info msg="[load/path_resolver] Paths resolving took 679.01µs: [.]"
level=info msg="[load] pkg github.com/go-openapi/errors files: [/home/go/src/github.com/go-openapi/errors/api.go /home/go/src/github.com/go-openapi/errors/auth.go /home/go/src/github.com/go-openapi/errors/doc.go /home/go/src/github.com/go-openapi/errors/headers.go /home/go/src/github.com/go-openapi/errors/middleware.go /home/go/src/github.com/go-openapi/errors/parsing.go /home/go/src/github.com/go-openapi/errors/schema.go /home/go/src/github.com/go-openapi/errors/api_test.go /home/go/src/github.com/go-openapi/errors/auth_test.go /home/go/src/github.com/go-openapi/errors/middleware_test.go /home/go/src/github.com/go-openapi/errors/parsing_test.go /home/go/src/github.com/go-openapi/errors/schema_test.go]"
level=info msg="[load] Program loading took 2.30193201s"
level=info msg="[load] SSA repr building took 640.489743ms"
level=info msg="[runner] worker.3 took 372.755474ms with stages: goimports: 98.702912ms, interfacer: 96.10616ms, dupl: 95.297143ms, ineffassign: 26.03004ms, deadcode: 20.508354ms, govet: 16.58933ms, varcheck: 14.596737ms, unconvert: 1.184018ms, errcheck: 1.070452ms, goconst: 625.767µs, structcheck: 589.065µs, gocyclo: 537.481µs, prealloc: 520.253µs, nakedret: 298.837µs, depguard: 4.784µs, typecheck: 3.374µs"
level=info msg="[runner] worker.4 took 402.954759ms with stages: golint: 150.803421ms, misspell: 115.734157ms, gosec: 74.367041ms, gofmt: 49.92ms"
level=info msg="[runner] worker.2 took 773.69374ms with stages: unparam: 773.680903ms"
level=info msg="[runner] worker.1 took 1.734878544s with stages: megacheck: 1.734871128s"
level=info msg="[runner] Workers idle times: #2: 960.887389ms, #3: 1.351649341s, #4: 1.331626828s"
level=info msg="[runner] processing took 14.16935ms with stages: path_prettifier: 13.521058ms, exclude: 440.298µs, autogenerated_exclude: 193.478µs, cgo: 6.559µs, nolint: 1.603µs, max_same_issues: 1.448µs, max_from_linter: 931ns, max_per_file_from_linter: 893ns, uniq_by_line: 860ns, source_code: 858ns, diff: 757ns, skip_files: 607ns"
level=info msg="Memory: 43 samples, avg is 320.2MB, max is 616.1MB"
level=info msg="Execution took 4.685521139s"
```

**Comment by jirfag** [comment]
Sorry for the slow response, now it's fixed

**Comment by fredbi** [comment]
Thank you @jirfag
